### ISSUE TITLE
Form field borders need a higher contrast

### ISSUE THREAD

**Issue by its-me-mario** [az-digital/arizona-bootstrap#1776]
## Problem/Motivation
Form input fields should have [at least a 3:1 contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#adjacent-colors).

### Describe the bug
Border colors are a little light.

### To Reproduce
Drop a form field on a page with a white background.

## Proposed resolution
Color would need to be updated to have at least a 3:1 contrast ratio
